config/blobs.yml

```diff
@@ -22,6 +22,10 @@ cf_exporter/cf_exporter_1.0.0.linux-amd64.tar.gz:
   size: 6880501
   object_id: c191fb50-f3aa-43fe-7514-c66f663fbdc5
   sha: sha256:303ac729668b9822f80da81e36ab5f83f53fa9a4c2a6e54075341649504082b9
```

Comment:
I think normally we remove the old blob, or?

Reply:
done